features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_ABNORMAL_REQUEST_COUNT.siddhi

```diff
@@ -44,22 +44,22 @@ define table ApimAllAlert (type string, tenantDomain string, message string, sev
 -- This aggregation definition is only for retrieving data. No data is actually aggregated from this.
 @store(type = 'rdbms', datasource = 'APIM_ANALYTICS_DB')
 define aggregation ApiUserPerAppAgg
-from Request 
-select apiName, apiVersion, apiContext, apiCreator, apiCreatorTenantDomain, apiHostname, applicationId, applicationName, applicationOwner, username, userTenantDomain, count(apiName) as totalRequestCount, gatewayType, label 
+from Request
+select apiName, apiVersion, apiContext, apiCreator, apiCreatorTenantDomain, apiHostname, applicationId, applicationName, applicationOwner, username, userTenantDomain, count() as totalRequestCount, gatewayType, label,'default' as regionalID
```

Comment:
```diff
--,'default' as regionalID
++ ,'default' as regionalID

Reply:
Fixed in 705b19ecd44e0d78cc4f30342c6d4c9c9d7c4337